pkg/allocation/converters/converter_test.go

```diff
@@ -366,7 +366,7 @@ func TestConvertAllocationRequestToGameServerAllocation(t *testing.T) {
 		},
 		{
 			name:     "empty fields to GSA with selectors fields",
-			features: fmt.Sprintf("%s=false&%s=false&%s=false", runtime.FeaturePlayerAllocationFilter, runtime.FeatureStateAllocationFilter, runtime.FeatureCountsAndLists),
```

Comment:
This test will need to be changed, since it had `TestConvertAllocationRequestToGameServerAllocation` set to false, so it's expecting a `nil` GameServer.